apps/vocabulary/v2Processor.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"io/ioutil"
+
+	"github.com/golang/protobuf/proto"
+
+	openapi_v2 "github.com/googleapis/gnostic/openapiv2"
+)
+
+func readDocumentFromFileWithNameV2(filename string) (*openapi_v2.Document, error) {
+	data, err := ioutil.ReadFile(filename)
+	if err != nil {
+		return nil, err
+	}
+	document := &openapi_v2.Document{}
+	err = proto.Unmarshal(data, document)
+	if err != nil {
+		return nil, err
+	}
+	return document, nil
+
+}
+
+func processDocumentV2(document *openapi_v2.Document, schemas map[string]int, operationId map[string]int, names map[string]int, properties map[string]int) {
```

Comment:
When there are sequential function arguments that share a type, they can be defined together like so:

func processDocumentV2(document *openapi_v2.Document, schemas, operationId, names, properties map[string]int)

Please do that here and elsewhere.

Reply:
Thanks for suggesting this! I'm going to address it in the next PR